hi-ml-multimodal/src/health_multimodal/image/model/model.py

```diff
@@ -98,6 +99,9 @@ def __init__(self,
         self.freeze_encoder = freeze_encoder
         self.train()
 
+        if weights == Weights.BIOVIL and pretrained_model_path is None:
```

Comment:
`get_biovil_resnet` from `image.model` does exactly this - why not use that function to load BioViL instead of adding logic to this class?

Reply:
We are currently using the `ImageModel` class directly, as this makes selecting random weights easier just by passing an argument directly from the config file to `ImageModel` when initialising. The alternative was calling `get_biovil_resnet` and then having to call an extra method to randomly re-initialise the model weights, which felt a bit messy...